protocols/virtualfund/virtualfund.go

```diff
@@ -600,10 +601,21 @@ func (o *Objective) acceptLedgerUpdate(c Connection, sk *[]byte) (protocols.Side
 	var signedProposal consensus_channel.SignedProposal
 
 	sideEffects := protocols.SideEffects{}
-	messages := protocols.CreateSignedProposalMessages(o.Id(), signedProposal, uint(ledger.MyIndex))
-	sideEffects.MessagesToSend = append(sideEffects.MessagesToSend, messages...)
+	message := o.createSignedProposalMessage(signedProposal, c.Channel)
+	sideEffects.MessagesToSend = append(sideEffects.MessagesToSend, message)
 	return sideEffects, nil
+}
 
+func (o *Objective) createSignedProposalMessage(sp consensus_channel.SignedProposal, ledger *consensus_channel.ConsensusChannel) protocols.Message {
```

Comment:
nit but could we add godoc to this

Reply:
216c4c4a720bfbadff170a5792e44ac57c3ab49f